business/istio_validations.go

```diff
@@ -148,6 +151,7 @@ func (in *IstioValidationsService) GetIstioObjectValidations(namespace string, o
 	wg.Add(9)
```

Comment:
I think you should increase this to 10.
The [working group](https://gobyexample.com/waitgroups) `wg` is used to manage the concurrence. Each `go routine` uses it for the sync `defer wg.Done()`. If there are more Done() than the expected, then you've got the following error:

```
panic: sync: negative WaitingGroup counter
```